### PR TITLE
support bert models and run tests successfully

### PR DESCRIPTION
**Pull request by Ldpe2G**
Support Bert Models for Oneflow backend for running the Taiyi-Stable-Diffusion.

### CODE REVIEW

File: src/transformers/modeling_oneflow_utils.py [hunk 1]
```diff
@@ -747,7 +747,7 @@ def create_extended_attention_mask_for_decoder(input_shape, attention_mask, devi
                     torch.ones((batch_size, seq_length, prefix_seq_len), device=device, dtype=causal_mask.dtype),
                     causal_mask,
                 ],
-                axis=-1,
+                dim=-1,
```
Comment (jackalcooper) [inline src/transformers/modeling_oneflow_utils.py:750]: 这里是oneflow的api没对齐吗

Comment (Ldpe2G) [reply src/transformers/modeling_oneflow_utils.py:750]: 是的，不知道为啥我们是 `dim ` 而不是 `axis`
